components/Article.jsx

```diff
@@ -1,46 +1,74 @@
-
-import { useParams } from 'react-router-dom';
-import { useState, useEffect } from 'react'
+import { useParams } from "react-router-dom";
+import { useState, useEffect } from "react";
 import axios from "axios";
 
+const Article = () => {
+  const [article, setArticle] = useState({});
+  const { article_id } = useParams();
+  const [comments, setComment] = useState([])
 
-const Article = ()=>{
-    const [article, setArticle] = useState({});
-    const {article_id } = useParams()
+  useEffect(() => {
+    axios
+      .get(`https://nc-news-api-4.onrender.com/api/articles/${article_id}`)
+      .then((response) => {
+       
+        setArticle(response.data.article);
+      })
+      .catch((error) => {
+        console.error("Error fetching the items:", error);
+      });
+  }, [article_id]);
 
+  useEffect(() => {
+    axios
+      .get(`https://nc-news-api-4.onrender.com/api/articles/${article_id}/comments`)
```

Comment:
This axios request (and the one above) should be packed away into a utils `api.js` file 👍